### PR TITLE
Fixed mistakes in building_blocks

### LINKED ISSUE #1401: Copyediting of documentation
## Enhancement proposal
As I look through the docs, I'll make a list here of pages that could do with some copyediting. Most of these pages are relatively well-written and only need some minor corrections. I think it'll be a good task for new contributors who are proficient with English.

- [ ] `installation/win`
- [ ] `installation/troubleshooting`
- [ ] `installation/for_dev` Last section seems incomplete
- [ ] `installation/plugins` Some commas can be semicolons
- [ ] `tutorials/building_blocks` "Mobjects are the basic building block**s**; probably other minor issues as well
- [ ] `tutorials/configuration` Some awkward phrasing
- [ ] `contributing/development` One `flake8` word isn't formatted correctly
- [ ] `contributing/internationalization` I copyedited this once as part of my review, but it could still do with another pass.

This is by no means an exhaustive list. Hopefully this helps new contributors with a keen eye for grammar and style find some ideas for ways to contribute.

## Additional comments
I was going to make a more exhaustive list, but I realized that almost every page in the documentation would be listed here if I were nitpicky enough.

### PR DESCRIPTION
**Pull request by mostlyaman**
## Overview: What does this pull request change?

Checked `building_blocks.rst`. Also pushing changes for other files, if I find any mistakes.

## Further Information and Comments

Closes #1401

## Reviewer Checklist
- [ ] The PR title is descriptive enough for the changelog, and the PR is labeled correctly
- [ ] If applicable: newly added non-private functions and classes have a docstring including a short summary and a PARAMETERS section
- [ ] If applicable: newly added functions and classes are tested

### CODE REVIEW

**Comment by Darylgolden** [comment]
Hi, is this ready for review?

**Comment by mostlyaman** [comment]
Hi, I apologise I have taken so much time, I have been busy and could not take time out to complete this PR. I suppose it's best for this PR to get reviewed and merged for now.

File: docs/source/tutorials/building_blocks.rst [hunk 1]
```diff
@@ -325,7 +325,7 @@ Suppose you are starting at 50 and incrementing until the :class:`~.DecimalNumbe
 * If alpha is 1, you want the value to be 100.
 
 Generally, you start with the starting number and add only some part of the value to be increment according to the alpha value.
-So, the logic of calculating the number to display at each step will be - 50 + alpha * (100 - 50).
+So, the logic of calculating the number to display at each step will be; 50 + alpha * (100 - 50).
```
[the remark this answers is not in the document]
Comment (Darylgolden) [reply docs/source/tutorials/building_blocks.rst:328]: I don't think semicolon is correct here. Leaving out the punctuation and formatting the formula as a code literal will probably be best.

**Review by Darylgolden — CHANGES_REQUESTED** [review]

**Review by Darylgolden — APPROVED** [review]
Thanks, this LGTM!

**Review by lucasricci — APPROVED** [review]
LGTM

**Pull request merged**